Added macros with alternative names; improved sizes and spacing with different kinds of brackets

ickc opened the pull request:
A continuation from #22.

This one should be clean from any development files.

pkra commented on physics/physics.js:
```diff
@@ -20,5 +20,4 @@
  *
- */
-MathJax.Hub.Register.StartupHook("TeX Jax Ready",function(){MathJax.InputJax.TeX.Definitions.Add({macros:{pqty:["Macro","{( #1 )}",1],bqty:["Macro","{[ #1 ]}",1],Bqty:["Macro","{\\{ #1 \\}}",1],abs:["Macro","{\\vert #1 \\vert}",1],norm:["Macro","{\\vert\\vert #1 \\vert\\vert}",1],eval:["Macro","{#1 \\vert}",1],order:["Macro","{\\mathcal{O} ( #1 )}",1],comm:["Macro","{[ #1 , #2 ]}",2],acomm:["Macr ...
+ */MathJax.Hub.Register.StartupHook("TeX Jax Ready",function(){MathJax.InputJax.TeX.Definitions.Add({macros:{quantity:["Macro","{	\\left\\{ #1 \\right\\}	}",1],qty:["Macro","{	\\left\\{ #1 \\right\\}	}",1],pqty:["Macro","{	\\left( #1 \\right)	}",1],bqty:["Macro","{	\\left[ #1 \\right]	}",1],vqty:["Macro","{	\\left\\vert #1 \\right\\vert	}",1],Bqty:["Macro","{	\\left\\{ #1 \\right\\}	}",1],absolut ...
```
This seems unintentional?

pkra commented on physics/README.md:
```diff
@@ -1,218 +1,72 @@
+---
+HTML Header:	<script type="text/x-mathjax-config">MathJax.Ajax.config.path.Contrib="https://cdn.mathjax.org/mathjax/contrib",MathJax.Hub.Register.StartupHook("TeX Jax Ready",function(){MathJax.Hub.Insert(MathJax.InputJax.TeX.Definitions.macros,{cancel:["Extension","cancel"],bcancel:["Extension","cancel"],xcancel:["Extension","cancel"],cancelto:["Extension","cancel"]})}),MathJax.Hub.Config({TeX:{e ...
```
Just wanted to double check if you want this header to show up; GitHub rendering doesn't handle it too well but that's your call.

pkra commented:
Thanks for this! I added two comments but otherwise this looks good.

ickc commented:
Hi, I updated the readme.

For the physics.js, do you mean there are a lot of tab characters there? I used uglifyjs2 and thought it would handled them. I cleaned them up just now.

pkra commented:
> For the physics.js, do you mean there are a lot of tab characters there?

Sorry for being incomplete earlier. I meant that the code seems commented out.

ickc commented:
I added a few tests on my github pages. It seems fine. Since the comment are closed `\*...*\`, it shouldn't comment out the following codes.

pkra commented:
D'oh -- my apologies! I hadn't looked carefully and was thrown off by GitHub's preview graying everything out. Thanks again for the great work!

pkra commented:
https://cdn.mathjax.org/mathjax/contrib/physics/physics.js and https://cdn.mathjax.org/mathjax/contrib/physics/unpacked/physics.js should be up-to-date now (might need clearing the browser cache)

pkra commented:
Thanks again @ickc !

ickc commented:
Thank you!
